tools/db/build/common.inc.php

```diff
@@ -7,7 +9,7 @@
   function cache($url, $filename, $duration = 60 * 60 * 24 * 7, $do_force = false) {
```

Comment:
$duration looks like 1 week (vs the comment "older than two weeks")

Reply:
hmm, that's an old comment then :)